From 05303c20db673ed5c64f3906d9db463e70dcf1da Mon Sep 17 00:00:00 2001
From: Robert Mader <robert.mader@collabora.com>
Date: Wed, 28 Jan 2026 15:15:30 +0100
Subject: [PATCH 5/6] aperture: Fix bitrate for openh264enc

The elements uses bits/sec, not kbits/sec like most other encoders.
---
 aperture/src/viewfinder.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/aperture/src/viewfinder.rs b/aperture/src/viewfinder.rs
index 265cc4f..64dfcd0 100644
--- a/aperture/src/viewfinder.rs
+++ b/aperture/src/viewfinder.rs
@@ -920,7 +920,7 @@ impl Viewfinder {
             )
             .item(
                 ElementPropertiesMapItem::builder("openh264enc")
-                    .field("bitrate", DEFAULT_BITRATE)
+                    .field("bitrate", DEFAULT_BITRATE * 1024)
                     .build(),
             )
             .item(
-- 
2.52.0

